docs/user_guide.md

```diff
@@ -36,72 +36,72 @@ a number of different grids, several routines are used to encode a message.
 This should give users more flexibility in how to organize data
 within one or more GRIB2 messages.
 
-To start a new GRIB2 message, call subroutine GRIBCREATE.  GRIBCREATE 
-encodes Sections 0 and 1 at the beginning of the message.  This routine 
+To start a new GRIB2 message, call subroutine GRIBCREATE(). GRIBCREATE()
+encodes Sections 0 and 1 at the beginning of the message. This routine 
 must be used to create each message.
 
-Subroutine ADDLOCAL can be used to add a Local Use Section ( Section 2 ).
+Subroutine ADDLOCAL() can be used to add a Local Use Section (Section 2).
```

Comment:
I see you added () to the subroutine names. However, doxygen did not create links to these routines.

Reply:
Do you know how to set it?
I add *.mod to the files, but it still not treat this file as code.

Reply:
I was looking at other repos for an example, but did not find one. Maybe it is not possible to create links to source code.

Reply:
Try lower-casing the function name...

Reply:
Lower-casing works.